packages/sakuli-legacy-types/src/build-dts-file.ts

```diff
@@ -1,15 +1,25 @@
 import {Application, DeclarationReflection} from 'typedoc';
 import {join} from "path";
 import {renderDeclaration} from "./template.function";
-import {writeFileSync, mkdirSync} from "fs";
-import { throwIfAbsent } from '@sakuli/commons';
+import {mkdirSync, writeFileSync} from "fs";
+import {throwIfAbsent} from '@sakuli/commons';
+import {Reflection, TraverseProperty} from "typedoc/dist/lib/models/reflections/abstract";
+import {ScriptTarget} from 'typescript';
 
 const outDir = 'dist';
 const projectDir = join(__dirname, '..', '..', 'sakuli-legacy');
 const srcDir = join(projectDir, 'src');
-const app = new Application({
+const app = new Application();
+app.bootstrap({
     tsconfig: join(projectDir, 'tsconfig.json'),
-    exclude: ['**/*.spec.ts', '**/__mocks__/*']
+    exclude: ['**/*.spec.ts', '**/__mocks__/*'],
+    mode: "file",
+    target: ScriptTarget.ES5,
```

Comment:
Target is ES2018 in typescript conf. Is the difference in the target specification okay? In addition, why do we have to add so much config to this file? :thinking: